physics/Interstitials/UFS_SCM_NEPTUNE/sfcsub.F

```diff
@@ -8163,6 +8339,18 @@ subroutine clima(lugb,iy,im,id,ih,fh,len,lsoil,slmskl,slmskw,     &
           vegclm(i) = wei1m * veg(i,k1) + wei2m * veg(i,k2)
         enddo
       endif
+
+!
+      if(fnlaic(1:8).ne.'        ') then
+        do i=1,len
+          laiclm(i) = wei1m * lai(i,k1) + wei2m * lai(i,k2)
+!         print '(A,F10.1)','laiclm=',laiclm(i)
```

Comment:
remove this commented print to be consistent with other similar calculations

Reply:
done